Sources/AppcuesKit/Data/Analytics/AnalyticsTracker.swift

```diff
@@ -93,10 +93,15 @@ internal class AnalyticsTracker: AnalyticsTracking, AnalyticsSubscribing {
             switch result {
             case .success(let qualifyResponse):
                 if #available(iOS 13.0, *), let experienceRenderer = self?.container?.resolve(ExperienceRendering.self) {
-                    experienceRenderer.show(
-                        qualifiedExperiences: qualifyResponse.experiences,
-                        priority: qualifyResponse.renderPriority,
-                        completion: nil)
+                    let experiments = qualifyResponse.experiments ?? [:]
+                    let qualifiedExperienceData = qualifyResponse.experiences.map {
+                        var experiment: Experiment?
+                        if let experimentID = $0.experimentID {
+                            experiment = experiments[experimentID]
+                        }
+                        return ExperienceData($0, priority: qualifyResponse.renderPriority, published: true, experiment: experiment)
+                    }
+                    experienceRenderer.show(qualifiedExperiences: qualifiedExperienceData, completion: nil)
```

Comment:
@mmaatttt now pre-processing the experiments here and bundling into `ExperienceData` to send into the `ExperienceRenderer` per our convo...